index.html

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8" />
+	<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+	<meta name="viewport" content="width=device-width, initial-scale=1" />
+	<link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet" />
+	<link rel="stylesheet" href="css/style.css" />
+	<script src="https://kit.fontawesome.com/102965b95f.js" crossorigin="anonymous"></script>
+	<title>Document</title>
+</head>
+
+<body>
+	<header class="header">
+		<nav class="bar">
+			<ul class="bar__cont">
+				<li class="bar__item"> <a class="bar__item" href="#"><i class="fa fa-twitter"></i> </a></li>
+				<li class="bar__item"><a class="bar__item" href="#"><i class="fa fa-facebook"></i></a></li>
+				<li class="bar__item"><a class="bar__item" href="#">English</a></li>
+				<li class="bar__item"><a class="bar__item" href="#">My Page</a></li>
+				<li class="bar__item"><a class="bar__item" href="#">Logout</a></li>
+			</ul>
+		</nav>
+		<i class="header__menu fa fa-bars fa-2x"></i>
+		<nav class="header__nav">
+			<a href="#home" class="header__home"> <img src="images/LogoMakr-7y5JLd.png" alt="Faith Apostolic logo"
+					class="header__logo"></a>
+			<ul class="header__nav--cont">
+				<li class="header__nav--item"><a href="#home">Home</a></li>
+				<li class="header__nav--item"><a href="about.html">About</a></li>
+				<li class="header__nav--item"><a href="#speakers">Speakers</a></li>
+				<li class="header__nav--item" id="button"><button type="button" class="header__camp">Campaign</button></li>
+			</ul>
+			
+		</nav>
+		<div class="mobile-menu" id="mobile-menu">
+			<button type="button" class="mobile-close">&times;</button>
+			<ul class="menu-list">
+				<li class="list-item"> <a href="#home">Home</a> </li>
+				<li class="list-item"><a href="about.html">About</a></li>
+				<li class="list-item"><a href="#speakers">Speakers</a></li>
+			</ul>
```

Comment:
- [ ] It'll be nice if you could reduce the font size used in the dropdown menu.

Reply:
@tenmarie. Thanks for your feedback but what font do you suggest matches the size of my main section. I tried to match what I already have so I don't want to make it too small. I am guessing this is an optional requirement. Thank you.

Reply:
Hey @the-catalystmc , yes it is optional, you could use a font size of maybe 25px